{{cookiecutter.project_shortname}}/scripts/server

```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env bash
 {% include 'misc/header.py' %}
-set -e
 
 script_path=$(dirname "$0")
-FLASK_ENV=development invenio run \
-    --cert "$script_path"/../docker/nginx/test.crt \
-    --key "$script_path"/../docker/nginx/test.key
+FLASK_ENV=development
+# Start Worker and send it to background
+pipenv run celery worker -A invenio_app.celery -l INFO & \
+    # Start server in parallel and wait for Ctrl+c
+pipenv run invenio run \
+       --cert "$script_path"/../docker/nginx/test.crt \
+       --key "$script_path"/../docker/nginx/test.key && \
+    # Once the worker is killed (Ctrl + c) kill also the server
+    kill $!
```

Comment:
I like lunching Celery and the Flask server at the same time, but again, my opinion, we can revert the changes

Reply:
I guess the only worry is that if the app crashes ungracefully the celery worker will keep running lost in the background... I don't know if some bash (or other) trick can help with that